fbw-a32nx/src/systems/instruments/src/MsfsAvionicsCommon/pixels.scss

```
@@ -1,3 +1,4 @@
+/* TO BE REMOVED TOGETHER WITH HOME COCKPIT MODE
```

```suggestion
/*  TODO FIXME: TO BE REMOVED TOGETHER WITH HOME COCKPIT MODE
```
prefix with TODO FIXME: